src/datasource.ts

```diff
@@ -209,7 +209,7 @@ export default class CogniteDatasource {
       return Promise.resolve([]);
     }
     return this.backendSrv.datasourceRequest({
-      url: this.url + `/cogniteapi/${this.project}/timeseries?q=${query}`,
+      url: this.url + `/cogniteapi/${this.project}/timeseries/search?query=${query}`,
```

Comment:
Doesn't this only work for projects where search has been enabled? If so, I'd prefer to not use it until it's enabled everywhere, and always, otherwise this plugin won't be "plug and play" for everyone.

I think it's not enabled for the "playground" project, for example.

Reply:
Fair enough.